test/e2e/config/multikueue/manager/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- ../../../../../dep-crds/training-operator/base/crds
```

Comment:
Can we use the `manifests/overlays/standalone/kustomization.yaml` file as a resource?

On this way we don't need to hardcode and replace `newTag`.

Reply:
not possible, let it stay like this for now

Reply:
Why?

Reply:
we want different behaviour in our kustomize, that excludes the possibility to use the one in original folder, thus we need to repeat whatever is there

Reply:
```suggestion
- ../../../../../dep-crds/training-operator/overlays/standalone
```

I mean this one.

Reply:
I checked this solution locally. Maybe I something missed?

Reply:
then maybe I tried it wrong, I will check

Reply:
but for the worker, for the manager we don't wan to install the operator
thus we point to base/crds